### ISSUE TITLE
NavigationView - Orientation Change

### ISSUE THREAD

**Issue by lolomu25** [mapbox/mapbox-navigation-ios#2364]
When changing the orientation of my device, I lose my position and when i get it back, it is off the road. How can I solve this issue.

![navigationissue](https://user-images.githubusercontent.com/2563427/80072115-d5de0380-8556-11ea-8aed-9bf6cf0fe384.gif)

**Comment by johnnewman** [comment]
A workaround you can implement is to call [`recenterMap`](https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation/NavigationMapView.swift#L1073) on the NavigationViewController's [`mapView`](https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation/NavigationViewController.swift#L88) instance when the device rotates.

Our NavigationViewController is embedded as a child view controller in a custom UIViewController. This approach worked for us in `viewWillTransition(to:with:)` within that parent view controller:
```Swift
override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
    super.viewWillTransition(to: size, with: coordinator)
        
    coordinator.animate(alongsideTransition: { (_) in
        self.mbNavigationViewController?.mapView?.recenterMap()
    })
}
```
You could probably subclass NavigationViewController and add the functionality that way, too.

**Comment by Udumft** [comment]
#2387 is merged in. If you cannot reproduce this using `master` branch - the ticket is done :)

**Comment by MaximAlien** [comment]
Thanks @Udumft, I've tested once again and no longer able to reproduce it. @lolomu25, if you have a chance please verify on your side whether you can reproduce this issue by building sources from `master`.

**Comment by 1ec5** [comment]
Fixed in #2387. Please let us know if you see this issue still. Thanks!
